css/theme/black.css

```diff
@@ -177,7 +177,9 @@ body {
 .reveal table td {
   text-align: left;
   padding: 0.2em 0.5em 0.2em 0.5em;
-  border-bottom: 1px solid; }
+  border-bottom: 1px solid;
```

Comment:
You'll need to change this in the underlying scss file otherwise it'll be overwritten whenever you run the `build` command.